assets/prototype/domain/eventEditor/datesList/dateCard/EditDateModal.js

```diff
@@ -0,0 +1,31 @@
+import { useContext } from '@wordpress/element';
+import FormModal from '../../../shared/FormModal';
+import DateForm from '../DateForm';
+import { DateTimeContext } from '../../../../infrastructure/services/contextProviders/DateTimeProvider';
+import useUpdateDateMutation from '../../containers/mutations/useUpdateDateMutation';
+
+const EditDateModal = () => {
+	const { id, isOpen, onClose, relatedTickets } = useContext(DateTimeContext);
+	const tickets = relatedTickets.map((id) => {
+		// we need to get here full ticket objects
+		return id;
+	});
```

Comment:
No need of that. All you need to do is pass those related IDs as the initial state in
`const [relatedItems, setRelatedItems] = useState([]);` in `RelationSelectr`

Reply:
Created a follow-up issue to address the `RelationSelector` issue: https://github.com/eventespresso/event-espresso-core/issues/1944